### PR TITLE
Calculator pr

### PR DESCRIPTION
**Pull request by ZeynebKoc**

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	const resetDisplay = () => {
+		display = '';
+	}
+	const deleteNum = () => {
+		display = display.slice(0, -1);
+	}
+	const equal = () => {
+		var exp = display;
+		if (exp){
+			display=eval(display);
```
Comment (erdemalpay) [inline src/App.svelte]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_eval!

Comment (ZeynebKoc) [reply src/App.svelte]: remove eval function

File: src/App.svelte [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		if(display.length <= 13){
+			display += String(num);	
+		}
+		operatorClicked = false;
+	}
+
+	const dot = () => {
+		if(display == 0) {
+			display = '0.';
+		}else if(!display.includes('.')){
```
Comment (asteyanetwork) [inline src/App.svelte]: There should be spaces after `}` and before `{`. There are also other places missing spaces. I suggest to use a linter like Eslint.

File: src/App.svelte [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+	const resetDisplay = () => {
+		display = 0;
+	}
+
+	const deleteNum = () => {
+		display = String(display);
+		if(display.length <= 1) { 
+			display = 0;
+		}else if (display == /.$/) {
+			display = display.slice(0, -2);
```
Comment (asteyanetwork) [inline src/App.svelte]: Why are you deleting 2 chars if it's ending with dot?

File: src/App.svelte [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+			display = 0;
+		}else if (display == /.$/) {
+			display = display.slice(0, -2); 
+		}else{
+			display = display.slice(0, -1);
+		}	
+	}	
+	
+	const equal = () => {
+		//debugger;
```
Comment (asteyanetwork) [inline src/App.svelte]: You should remove this if you don't need anymore.

File: src/App.svelte [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+		secondOperand = Number(display)
+		if(operator === '+'){
+		 	display = firstOperand + secondOperand;
+			display = display.toFixed(7).replace(/0+$/, '');
```
Comment (asteyanetwork) [inline src/App.svelte]: You can create a helper function above doing the job in this line to prevent repeating it 4 times.

File: src/App.svelte [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+		 	display =  firstOperand * secondOperand;
+			display = display.toFixed(7).replace(/0+$/, '');
+		}else if(operator === '/'){
+		 	display =  firstOperand / secondOperand;
+			display = display.toFixed(7).replace(/0+$/, '');
+		}
+	} 
+
+	const operationsState = (operation) => {
+		if(secondOperand, operatorClicked = true){
```
Comment (asteyanetwork) [inline src/App.svelte]: This seems very suspicious. `operatorClicked = true` would always return true. You are not comparing with `==` or `===` there. You are assigning with `=`. Also what's the point of secondOperand before comma there?
Did you mean if both secondOperand exists and operatorClicked equals to true? If so you should use `&&` operator between them.

**Review by asteyanetwork — CHANGES_REQUESTED** [review]
Good job 👍 . I have a few comments and request changes.

File: src/App.svelte (hunk 4 above)
Comment (ZeynebKoc) [reply src/App.svelte]: Removed

File: src/App.svelte (hunk 5 above)
Comment (ZeynebKoc) [reply src/App.svelte]: I fixed it by creating setFloatNumbers function.

File: src/App.svelte (hunk 6 above)
Comment (ZeynebKoc) [reply src/App.svelte]: I wanted to set true for both by putting a comma after the secondOperator. I couldn't run the function correctly using the && operator. So I moved operatorClicked up and rearranged the function by defining secondOperator as Number.

File: src/App.svelte (hunk 3 above)
Comment (ZeynebKoc) [reply src/App.svelte]: I added this to the wrong function. I fixed it.

File: src/App.svelte (hunk 2 above)
Comment (ZeynebKoc) [reply src/App.svelte]: Improved code readability

**Pull request merged**